cli/cmd/network.go

```diff
@@ -11,10 +12,13 @@ import (
 
 func (r *runners) InitNetworkCommand(parent *cobra.Command) *cobra.Command {
 	cmd := &cobra.Command{
-		Use:    "network",
-		Short:  "Manage test networks for VMs and Clusters",
-		Long:   ``,
-		Hidden: true,
+		Use:   "network",
+		Short: "Manage test networks for VMs and Clusters",
+		Long: `The 'network' command allows you to manage and interact with networks used for testing purposes.
+With this command you can list the networks in use by VMs and clusters.`,
+		Example: `# List all networks
+replicated network ls`,
```

Comment:
`network ls` columns are going to show `REPORTING` and `POLICY` even though those are airgap features.  In the past this wasn't a major issue for Amber (like on the CMX history page), but it might be worth double checking that that's ok with product.